lib/api/api.js

```diff
@@ -174,6 +175,9 @@ const api = {
             if (userInfo.isRequesterAnIAMUser()) {
                 authNames.userName = userInfo.getIAMdisplayName();
             }
+            if (isRequesterASessionUser(userInfo)) {
+                authNames.sessionName = userInfo.getShortid().split(':')[1];
```

Comment:
Do we use this `sessionName` somewhere?

Reply:
no, actually it's just for log printing.